chapters/funkcije/funkcije.tex

```diff
@@ -161,28 +169,25 @@ \section{Rekurzija}
 sebe. Pogosto lahko zanko nadomestimo z rekurzijo, če nam je tak način
 programiranja ljubši.
 
-Recimo, da želimo sešteti vsa števila med 1 in nekim $n$. Z zanko bi to
-naredili tako, da si pripravimo neko srpemenljivko $vsota$ in potem napišemo
-for zanko od 1 do n in v $vsota$ seštejemo vsa ta števila. Pri rekurziji je pa
-pristop drugačen.
+Recimo, da želimo sešteti vsa števila med 1 in nekim \verb+n+. Z zanko bi to
+naredili tako, da si pripravimo neko spremenljivko \verb+vsota+ in potem
+napišemo for zanko od 1 do \verb+n+ in v \verb+vsota+ seštejemo vsa ta števila. Pri
+rekurziji je pa pristop drugačen.
 
 \begin{examples}
     %#insert python3 style.py < chapters/funkcije/vsota_rekurzivno.cpp
 \end{examples}
 
 Najprej napišemo \emph{osnovni pogoj}, ki za n-je manjše ali enake 0 vrne 0. Če
-ne bi napisali osnovnega pogoja bi šli rekurzivni klici funkcije v neskončnost,
+ne bi napisali osnovnega pogoja bi, šli rekurzivni klici funkcije v neskončnost,
 \verb+vsota(3)+ bi poklicala \verb+vsota(2)+, ki pokliče \verb+vsota(1)+, potem
 \verb+vsota(0)+, \verb+vsota(-1)+, \verb+vsota(-2)+, \ldots \ Po osnovnem
 pogoju moramo narediti \emph{korak}, ki izračuna \verb+vsota(n)+ s pomočjo
-vrednost funkcije v nekem manjšem številu, v našem primeru kar
+vrednosti funkcije v nekem manjšem številu, v našem primeru kar
 \verb+vsota(n - 1)+.
 
-O rekurziji bi lahko veliko povedali in je za reševanje raznih problemov zelo
-uporabna, a smo se odločili, da se letos o njej ne bomo poglabljali.
-
 
-\section{Funkcije že vključene v C}
+\section{Funkcije v C knjižnici}
 
 \subsection{Funkcije iz <stdio.h>}
```

Comment:
Tu (in spodaj pri `<string.h>` bi bilo lepo dati `\texttt` (se opravičujem za prej, nisem vedel, da `\verb` ne gre v naslove)